### DISCUSSION TITLE
how to change the cli prompt char '>' to something else?

### DISCUSSION THREAD

**Discussion by NizanMandelblit** [funbiscuit/embedded-cli#20] [Q&A] [answered]
how to change the cli prompt char '>' to something else?

**Comment by NizanMandelblit** [comment, marked as answer]
found it!
impl->invitation at line 793.

**Reply by funbiscuit** [reply]
Yes, but that's internals of the lib. Looks like it should be moved to config. I'll create an issue.

**Comment by funbiscuit** [comment]
Since #28 you can change invitation to anything else via config:
```C
EmbeddedCliConfig *config = embeddedCliDefaultConfig();
config->invitation = "::> ";
```
